packages/orbit-components/src/Collapse/Collapse.ct-story.tsx

```diff
@@ -19,9 +19,15 @@ export default function CollapseStory() {
       >
         <p>Collapsed content</p>
       </Collapse>
-      <Collapse label="Expanded" initialExpanded>
+      <Collapse label="Initial expanded" initialExpanded>
+        <p>Initial expanded content</p>
+      </Collapse>
+      <Collapse label="Expanded" expanded>
         <p>Expanded content</p>
       </Collapse>
+      <Collapse label="Collapsed">
+        <p>Content</p>
+      </Collapse>
```

Comment:
Why do we need these new Collapses?

My idea was to rely on the existing tests with actions, so we're sure nothing changed visually after the component refactor. Perhaps I'm missing some details.

Reply:
My idea was to cover cases when `Collapse` is clicked (case for click on collapse wrapper, case for click on collapse arrow button). Before that, we had visual snapshots of expanded collapses. Do you think these cases are unnecessary?

Reply:
Hmm, I guess those cases are better covered by unit tests instead, what do you think?

If we keep visual tests as they were, we can make sure that the refactor is not affecting the component visually, which is very helpful.

Reply:
Ok, I agree. I'm reverting those test back. ♻️